tgg/seqr_scripts/sex_check.py

```diff
@@ -0,0 +1,340 @@
+    :param y_cov_threshold: Coverage on chrY above which supports male call. Default is 0.1
+    :param normalization_contig: Chosen chromosome for calculating normalized coverage. Default is "20"
+    :param xy_fstat_threshold: F-stat threshold above which a sample will be called XY. Default is 0.75
+    :param xx_fstat_threshold: F-stat threshold below which a sample will be called XX. Default is 0.5
+    :param aaf_threshold: Alternate allele frequency threshold for `hl.impute_sex`. Default is 0.05
+    :param call_rate_threshold: Minimum required call rate. Default is 0.25
+    :return: Table with sex annotations
+    """
+
+    # Read in matrix table and define output directory
+    # TODO: Generalize before moving into gnomad_methods
+    outdir = dirname(callset)
+    mt_name = callset.split("/")[-1].strip("\.mt")
+    logger.info("Reading matrix table for callset: %s", callset)
+    logger.info("Using chromosome Y coverage? %s", use_y_cov)
+
+    mt = hl.read_matrix_table(callset)
+
+    # Filter to SNVs and biallelics
+    mt = mt.filter_rows(
+        (hl.len(mt.alleles) == 2) & hl.is_snp(mt.alleles[0], mt.alleles[1])
+    )
+
+    # Filter to pass variants only (empty set)
+    # TODO: Make this an optional argument before moving to gnomad_methods
+    mt = mt.filter_rows(mt.filters.length() == 0, keep=True)
+
+    # Infer build:
+    build = get_reference_genome(mt.locus).name
+    logger.info("Build inferred as %s", build)
+
+    logger.info("Inferring sex...")
+    # TODO: Change "female" and "male" to "XX" and "XY"
+    if use_y_cov:
+        sex_ht = get_chr_cov(mt, "GRCh38", normalization_contig)
+        mt = mt.annotate_cols(**sex_ht[mt.col_key])
+        sex_ht = get_chr_cov(mt, "GRCh38", "Y")
+        mt = mt.annotate_cols(**sex_ht[mt.col_key])
+        mt = mt.annotate_cols(
+            normalized_y_coverage=hl.or_missing(
+                mt[f"chr{normalization_contig}_mean_dp"] > 0,
+                mt.chrY_mean_dp / mt[f"chr{normalization_contig}_mean_dp"],
+            )
+        )
+        sex_ht = run_hails_impute_sex(
+            mt,
+            build,
+            outdir,
+            mt_name,
+            xy_fstat_threshold,
+            xx_fstat_threshold,
+            aaf_threshold,
+        )
+        sex_ht = sex_ht.annotate(
+            ambiguous_sex=hl.is_missing(sex_ht.is_female),
+            sex_aneuploidy=(sex_ht.is_female)
+            & hl.is_defined(sex_ht.normalized_y_coverage)
+            & (sex_ht.normalized_y_coverage > y_cov_threshold)
+            | (~sex_ht.is_female)
+            & hl.is_defined(sex_ht.normalized_y_coverage)
+            & (sex_ht.normalized_y_coverage < y_cov_threshold),
+        )
+
+        sex_expr = (
+            hl.case()
+            .when(sex_ht.ambiguous_sex, "ambiguous_sex")
+            .when(sex_ht.sex_aneuploidy, "sex_aneuploidy")
+            .when(sex_ht.is_female, "female")
+            .default("male")
```

Comment:
```suggestion
            .when(sex_ht.is_female, "XX")
            .default("XY")
```
we are trying to change the language we use within our group; "XX" and "XY" are biological sexes, but "female" and "male" are genders